starters/express-apollo-prisma/src/graphql/schema/technology/technology.typedefs.ts

```diff
@@ -7,8 +7,12 @@ export const technologyTypeDefs = gql`
 	type Technology {
 		"The ID of the Technology"
 		id: ID!
-		"The name of the Technology"
-		name: String!
+		"The display name of the Technology"
+		displayName: String!
+		"The brief description of the Technology"
```

Comment:
```suggestion
		"A brief description of the Technology"
```
Sounds better i think :thinking: